.eslintrc.js

```diff
@@ -147,7 +140,9 @@ module.exports = {
     'no-invalid-this': 'off',
     '@babel/no-invalid-this': 'error',
 
-    '@babel/semi': ['error', 'never'],
+    semi: 'off',
+    '@babel/semi': 'error',
```

Comment:
I recommend turning this off... prettier will throw errors for issues with semicolons and will automatically fix with --fix

Reply:
Hm, do you know if `prettier` handle class properties? See `@babel/semi` comment in readme here: https://github.com/babel/babel/tree/main/eslint/babel-eslint-plugin

Reply:
it's turned off in eslint-config-prettier, so i'd assume so. https://github.com/prettier/eslint-config-prettier/blob/main/babel.js

Reply:
Done in: d5717e1